tracer/test/snapshots/AspNetMvc4Tests.Integrated.WithExpansion.__path=_Home_BadRequestWithStatusCode-statuscode=401&TransferRequest=true_statusCode=401.verified.txt

```diff
@@ -55,11 +59,15 @@
 System.Exception: Oops, it broke. Specified status code was: 401
 at Samples.AspNetMvc4.Controllers.HomeController.BadRequestWithStatusCode(Int32 statuscode),
       error.type: System.Exception,
+      http.client_ip: 127.0.0.1,
       http.method: GET,
       http.request.headers.host: localhost:00000,
+      http.route: /error/index,
```

Comment:
This is incorrect, this should be `{controller}/{action}/{id}` from the SpanId: Id_5. I'm guessing it's a weird interplay between the TransferRequest operation and the `WithExpansion` scenario...

Reply:
Okay I see what's going on, this is DEFINITELY a combination of those two scenarios.
- `WithExpansion` scenario issue: You can tell it's borked because the `http.route` does not take the `aspnet.route` value in the snapshot file `tracer/test/snapshots/AspNetMvc4Tests.Integrated.WithExpansion.__path=_Home_BadRequest_statusCode=500.verified.txt`.
- TransferRequest issue: This is exhibiting the exact behavior I described in [this comment](https://github.com/DataDog/dd-trace-dotnet/pull/2915#discussion_r922528378). I'll go ahead and resolve that, leaving us with this one issue to keep track of.

Reply:
My thinking is that `WithExpansion` should only affect the resource name and not the `http.route` tag. Do you all agree?

Reply:
fixed in 4d2355566ff23e3458e03b9e8a67f86b3c158b29